src/backend/distributed/commands/multi_copy.c

```diff
@@ -1075,23 +1156,27 @@ SendCopyBinaryFooters(CopyOutState copyOutState, List *connectionList)
  * shard.
  */
 static StringInfo
-ConstructCopyStatement(CopyStmt *copyStatement, int64 shardId)
+ConstructCopyStatement(CopyStmt *copyStatement, int64 shardId, bool binaryCopyFormat)
 {
 	StringInfo command = makeStringInfo();
 
-	char *schemaName = copyStatement->relation->schemaname;
-	char *relationName = copyStatement->relation->relname;
-
-	char *shardName = pstrdup(relationName);
+	char *shardSchemaName = copyStatement->relation->schemaname;
+	char *shardTableName = copyStatement->relation->relname;
 	char *shardQualifiedName = NULL;
 
-	AppendShardIdToName(&shardName, shardId);
-
-	shardQualifiedName = quote_qualified_identifier(schemaName, shardName);
+	shardTableName = pstrdup(shardTableName);
```

Comment:
Why not do this on the same line as initialization? If this were a more complicated function it'd make sense to separate it out for debuggability but I don't think future devs are likely to want to know the argument pstrdup was given.